service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsService.java

```diff
@@ -42,17 +45,25 @@
 
     public boolean hasEmptyDates(CaseData caseData) {
         return Stream.of(caseData.getAllParties(),
-                caseData.getLocalAuthorityDirections(),
-                caseData.getRespondentDirections(),
-                caseData.getCafcassDirections(),
-                caseData.getOtherPartiesDirections(),
-                caseData.getCourtDirections())
+            caseData.getLocalAuthorityDirections(),
+            caseData.getRespondentDirections(),
+            caseData.getCafcassDirections(),
+            caseData.getOtherPartiesDirections(),
+            caseData.getCourtDirections())
             .flatMap(Collection::stream)
             .map(Element::getValue)
             .map(Direction::getDateToBeCompletedBy)
             .anyMatch(Objects::isNull);
     }
 
+    public Map<String, List<Element<Direction>>> populateStandardDirections(CaseData caseData) {
```

Comment:
can be
`  public Map<String, List<Element<Direction>>> populateStandardDirections(CaseData caseData) {
        return getAssigneeToDirectionMapping(getDirections(caseData.getFirstHearing().orElse(null)))
            .entrySet().stream()
            .collect(Collectors.toMap(pair -> pair.getKey().getValue(), Map.Entry::getValue));
    }`